app/viewsets.py

```diff
@@ -0,0 +1,145 @@
+import os
+
+from django.shortcuts import get_object_or_404
+from django.conf import settings
+from django.core.files import File
+from django.core.files.uploadedfile import InMemoryUploadedFile
+
+from rest_framework import viewsets, permissions, status
+from rest_framework.response import Response
+
+from faker import Factory
+
+from .serializers import PhotoSerializer, EffectSerializer
+from .models import Photo, Effects, FILTERS
+from .permissions import IsOwner
+
+
+class EffectViewSet(viewsets.ModelViewSet):
+	"""Handle CRUD requests to '/effects/' url."""
```

Comment:
**PEP8:** Same to this, the closing `"""` should be on it's own line. Correct this and the rest.